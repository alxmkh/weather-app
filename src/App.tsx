import React from 'react';
import "antd/dist/antd.css";
import {Card, Col, Row, Space} from "antd";
import { Complete } from './Components/InputData/InputData';
import { DisplayData } from './Components/DisplayWeather/DisplayData';
import Title from "antd/lib/typography/Title";

const App: React.FC<any> = () => {
    return (
            <Row>
                <Col span={12} offset={8}>
                    <Space direction="vertical">
                        <Card title={<Title level={3}>WeatherApp</Title>} style={{ width: 400}}>
                            <div style={{paddingBottom:20}}>
                                <Complete />
                            </div>
                            <DisplayData/>
                        </Card>
                    </Space>
                </Col>
            </Row>
    );
}

export default App;
