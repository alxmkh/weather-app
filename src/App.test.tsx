import React from 'react';
import App from './App';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';
import Enzyme, { shallow, render, mount } from 'enzyme';


Enzyme.configure({adapter: new Adapter()});

describe(`Test '<App/> component.`, () => {
    it('Renders <App/> component and equal with snapshot.', () => {
        const tree = shallow(<App/>);
        expect(toJson(tree)).toMatchSnapshot();
    })
})


