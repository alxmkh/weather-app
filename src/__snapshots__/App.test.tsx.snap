// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test '<App/> component. Renders <App/> component and equal with snapshot. 1`] = `
<Row>
  <Col
    offset={8}
    span={12}
  >
    <Space
      direction="vertical"
    >
      <Card
        style={
          Object {
            "width": 400,
          }
        }
        title={
          <Title
            level={3}
          >
            WeatherApp
          </Title>
        }
      >
        <div
          style={
            Object {
              "paddingBottom": 20,
            }
          }
        >
          <Memo() />
        </div>
        <DisplayData />
      </Card>
    </Space>
  </Col>
</Row>
`;
